tools/src/main/java/com/hedera/block/tools/commands/record2blocks/model/SignatureFile.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.tools.commands.record2blocks.model;
+
+import java.util.HexFormat;
+
+/**
+ * SignatureFile represents a Hedera record file signature file.
+ * The below table describes the content that can be parsed from a record signature file.
+ *
+ * <table>
+ * <caption>Signature File Format</caption>
+ * <thead>
+ * <tr>
+ * <th><strong>Name</strong></th>
+ * <th><strong>Type (Bytes)</strong></th>
+ * <th><strong>Description</strong></th>
+ * </tr>
+ * </thead>
+ * <tbody>
+ * <tr>
+ * <td>File Hash Marker</td>
+ * <td>byte</td>
+ * <td>Value: 4</td>
+ * </tr>
+ * <tr>
+ * <td>File Hash</td>
+ * <td>byte[48]</td>
+ * <td>SHA384 hash of corresponding *.rcd file</td>
+ * </tr>
+ * <tr>
+ * <td>Signature Marker</td>
+ * <td>byte</td>
+ * <td>Value: 3</td>
+ * </tr>
+ * <tr>
+ * <td>Length of Signature</td>
+ * <td>int (4)</td>
+ * <td>Byte size of the following signature bytes</td>
+ * </tr>
+ * <tr>
+ * <td>Signature</td>
+ * <td>byte[]</td>
+ * <td>Signature bytes</td>
+ * </tr>
+ * </tbody>
+ * </table>
+ *
+ * @param fileHash SHA384 hash of corresponding *.rcd file
+ * @param signature Signature bytes or RSA signature of the file hash, signed by the node's private key
+ */
+public record SignatureFile(byte[] fileHash, byte[] signature) {
```

Comment:
Xin brought up a good point in the HIP review.
We probably need to track _which_ node this signature is for, and include that in the bytes stored in the record file item.  Otherwise we'll have no way to verify signatures on those items in the block stream because we won't know which node's public key to use.

Also, we'll need some way to populate what address book to use (unless we have some other source available for that).